R/cnd-message.R

```diff
@@ -45,13 +45,54 @@
 #'
 #' @export
 cnd_message <- function(cnd) {
-  paste_line(
+  cnd_format <- cnd_formatter(cnd)
+  cnd_format(cnd_message_lines(cnd))
+}
+cnd_message_lines <- function(cnd) {
+  c(
     cnd_header(cnd),
     cnd_body(cnd),
     cnd_footer(cnd)
   )
 }
 
+cnd_formatter <- function(cnd) {
```

Comment:
This is where the callback to cli happens for conditions that have `use_cli_format` set. The `indent` parameter is used in the display of chained errors.

TODO: Inform cli about the starting column which may vary depending on the context. Needs r-lib/cli#345.